tests/test_integration.py

```diff
@@ -455,6 +457,36 @@ def test_info(self):
         self.vm.shutdown()
         host_info(VM_HOSTNAME)
 
+    @patch('igvm.vm.VM.vm_performance_value', return_value=5.0)
+    def test_igvm_migration_log(self, mock_vm_performance_value):
+        self.vm.hypervisor.dataset_obj['igvm_migration_log'].clear()
+        self.vm.hypervisor.dataset_obj.commit()
+        self.assertFalse(
+            bool(self.vm.hypervisor.dataset_obj['igvm_migration_log'])
+        )
+
+        cpu_usage_vm = self.vm.hypervisor.hv_predict_vm_cpu_util(self.vm)
+        timestamp = int(time())
+        self.vm.hypervisor.hv_add_migration_log(self.vm, '-')
+        self.assertEqual(
+            list(self.vm.hypervisor.dataset_obj['igvm_migration_log']),
+            ['{} -{}'.format(timestamp, round(cpu_usage_vm))])
```

Comment:
If hv_add_migration_log takes more than one second, the timestamp will be different.

Reply:
timestamp is now mocked